lib/libnxpse050/adaptors/utils/info.c

```diff
@@ -0,0 +1,222 @@
+// SPDX-License-Identifier: BSD-3-Clause
+/*
+ * Copyright (C) Foundries Ltd. 2020 - All Rights Reserved
+ * Author: Jorge Ramirez <jorge@foundries.io>
+ */
+
+#include <fsl_sss_se05x_apis.h>
+#include <global_platf.h>
+#include <se050.h>
+#include <se05x_const.h>
+#include <se05x_tlv.h>
+#include <smCom.h>
+#include <string.h>
+
+/* these macros should use NX_LEVEL_INFO but due to a problem in the stack
+ * - excesive verbosity - we are bypassing while forcing the console (hence the
+ * 0xff). Once the stack is fixed we can use the NX values.
+ */
+
+#define LOG_MAU8_I(msg, buf, len) nLog_au8("Info", 0xff, msg, buf, len)
+
+#define LOG_I(format, ...) nLog("Info", 0xff, format, ##__VA_ARGS__)
+
+#define LOG_E(format, ...) nLog("Info", NX_LEVEL_ERROR, format, ##__VA_ARGS__)
```

Comment:
could these redicrect to `xMSG()` macros?

Reply:
the log info generated this way is better structured and organized IMO (the implementation is in wraps.c)

Reply:
As you wish. It somewhat duplicates what's done in **trace.h**.